lockfile/Cargo.toml

```diff
@@ -4,7 +4,7 @@ description = "Package lock file parsers used by Phylum"
 version = "0.1.0"
 authors = ["Phylum, Inc. <engineering@phylum.io>"]
 edition = "2021"
-rust-version = "1.60"
+rust-version = "1.62.1"
```

Comment:
I'm not sure how we should handle this... I think the `lockfile` crate would build just fine with 1.61 (or maybe even 1.60).

Since this is a library, we are restricting our users by putting a MSRV that is higher than necessary here. Maybe this crate shouldn't have an MSRV?

Reply:
I don't think not providing a MSRV guarantee at all is even worse tbh.

Initially I thought about separating the two MSRVs with two separate CI tests, I can go back to that if you'd prefer it? I just thought it would be unnecessarily complicated to do so.

Reply:
I agree that two separate tests is unnecessarily complicated.

I guess keeping it the same as the `phylum-cli` crate is fine for now